acme_diags/driver/enso_diags_driver.py

```diff
@@ -54,15 +54,26 @@ def calculate_nino_index_model(data, nino_region_str, parameter):
     """
 
     try:
-        sst = data.get_timeseries_variable('SST')
+        try:
+            # Try sea surface temperature first.
+            sst = data.get_timeseries_variable('SST')
+        except RuntimeError as e:
+            if str(e).startswith('Neither does SST nor the variables in'):
+                print(e)
```

Comment:
In the two `print(e)` lines it might be good to do `print('Handling exception=", e)` instead, to make it clear the user doesn't need to be overly concerned.